src/main/java/com/gregtechceu/gtceu/data/recipe/CustomTags.java

```diff
@@ -125,6 +125,7 @@ public class CustomTags {
     public static final TagKey<Block> CONCRETE_POWDER_BLOCK = TagUtil.createBlockTag("concrete_powders");
     public static final TagKey<Block> CREATE_SEATS = TagUtil.optionalTag(Registries.BLOCK,
             new ResourceLocation(GTValues.MODID_CREATE, "seats"));
+    public static final TagKey<Item> DOUGHS = TagUtil.optionalTag(Registries.ITEM, new ResourceLocation("forge:dough"));
```

Comment:
move this to be with the other item tags, and do `TagUtil.createItemTag("dough", false)` instead.